client/src/components/layouts/header/Header.tsx

```diff
@@ -1,5 +1,14 @@
+import "./Header.css";
+
 const Header = () => {
-  return <span>Header</span>;
+  return (
+    <header className="header">
+      <div className="header-right">
+        <button type="button">Langage</button>
+        <button type="button">Se connecter</button>
```

Comment:
Est-ce que "se connecter" redirige vers une autre page ?

Si oui, ce devrait être un élément `Link to="/login"`.

Quant au bouton `Langage`, quelle est sa vocation ? Ca ouvrirait une modale permettant de sélectionner une langue ?